apps/webapp/src/app/document-title/use-document-title.tsx

```diff
@@ -0,0 +1,10 @@
+import { Meeting } from '../meeting/meeting';
+import { useEffect, useState } from 'react';
+
+export const useDocumentTitle = (meeting: Meeting | undefined): string => {
```

Comment:
jakoś mi nie leży ten hook powiązany z meeting. ale jeśli faktycznie zrobisz title w stanie z efektem do synchronizacji skutków ubocznych, to teb hook zniknie, si?

Reply:
tak w sumie będzie